api/extensions/ext_redis.py

```diff
@@ -71,6 +72,13 @@ def init_app(app):
         )
         master = sentinel.master_for(dify_config.REDIS_SENTINEL_SERVICE_NAME, **redis_params)
         redis_client.initialize(master)
+    elif dify_config.REDIS_USE_CLUSTERS:
+        startup_nodes = [
+            {"host": node.split(":")[0], "port": int(node.split(":")[1])} 
+            for node in dify_config.REDIS_CLUSTERS.split(",")
+        ]
+        nodes = [ClusterNode(host=node["host"], port=node["port"]) for node in startup_nodes]
```

Comment:
```suggestion
        nodes = [
            ClusterNode(host=node.split(":")[0], port=int(node.split(":")[1])) 
            for node in dify_config.REDIS_CLUSTERS.split(",")
        ]
```
Simplified version without repeated looping.

Reply:
Okay, I have submitted the code according to your suggestions